### PR TITLE
SQLite connection and source files fixes, removed fails ignoring

### PR DESCRIPTION
**Pull request by AlexanderSashchenko**

### CODE REVIEW

File: src/main/groovy/liquibase/harness/util/TestUtils.groovy [hunk 1]
```diff
@@ -46,6 +47,9 @@ class TestUtils {
         try {
             commandScope.execute()
         } catch (Exception exception) {
+            if (exception instanceof CommandExecutionException && exception.toString().contains("is not available in SQL output mode")) {
```
Comment (KushnirykOleh) [inline src/main/groovy/liquibase/harness/util/TestUtils.groovy:50]: not obvious why we are doing this, would be nice to create separate method for handling it (as check may expand in future) or at least add comment

File: src/test/resources/sqlite/sqlite-init.txt [hunk 2]
```diff
@@ -0,0 +1,28 @@
+DROP TABLE IF EXISTS authors;
```
Comment (KushnirykOleh) [inline src/test/resources/sqlite/sqlite-init.txt:1]: make this .sql file

**Review by KushnirykOleh — CHANGES_REQUESTED** [review]

File: src/main/groovy/liquibase/harness/util/TestUtils.groovy (hunk 1 above)
Comment (AlexanderSashchenko) [reply src/main/groovy/liquibase/harness/util/TestUtils.groovy:50]: fixed

File: src/test/resources/sqlite/sqlite-init.txt (hunk 2 above)
Comment (AlexanderSashchenko) [reply src/test/resources/sqlite/sqlite-init.txt]: fixed

**Review by KushnirykOleh — APPROVED** [review]

**Pull request merged**